src/SwedbankPay/Api/Client/ClientVersion.php

```diff
@@ -0,0 +1,197 @@
+      * @return bool true if successful; otherwise false.
+      */
+    private function tryGetVersionNumberFromConstant(&$version) : bool
+    {
+        if (defined('VERSION') && !is_null(VERSION) && !empty(VERSION)) {
+            $version = VERSION;
+            return true;
+        }
+
+        return false;
+    }
+
+
+    /**
+      * Tries to get the version number from the environment variable VERSION.
+      * Returns true if successful; otherwise false.
+      *
+      * @param string $version The by-reference $version variable to assign the version number to, if found.
+      * @return bool true if successful; otherwise false.
+      */
+    private function tryGetVersionNumberFromEnv(&$version) : bool
+    {
+        // phpcs:disable
+        $envVersion = getenv("VERSION");
+        // phpcs:enable
+
+        if ($envVersion !== false && !is_null($envVersion) && !empty($envVersion)) {
+            $version = $envVersion;
+            return true;
+        }
+
+        return false;
+    }
+
+
+    /**
+      * Tries to get the version number from the composer.json file.
+      * Returns true if successful; otherwise false.
+      *
+      * @param string $version The by-reference $version variable to assign the version number to, if found.
+      * @return bool true if successful; otherwise false.
+      */
+    private function tryGetVersionNumberFromComposerJson(&$version) : bool
+    {
+        $composerPath = __DIR__ . '/../../../../composer.json';
+        $composer = null;
+
+        if (!$this->tryReadJson($composerPath, $composer)) {
+            return false;
+        }
+
+        if (isset($composer['version'])) {
+            $version = $composer['version'];
+
+            if (!is_null($version) && !empty($version)) {
+                return true;
+            }
+        }
+
+        return false;
+    }
+
+
+    /**
+      * Tries to get the version number from the composer.lock file.
+      * Returns true if successful; otherwise false.
+      *
+      * @param string $version The by-reference $version variable to assign the version number to, if found.
+      * @return bool true if successful; otherwise false.
+      */
+    private function tryGetVersionNumberFromComposerLock(&$version) : bool
+    {
+        $composerLockPath = getcwd() . DIRECTORY_SEPARATOR . 'composer.lock';
+        $composerLock = null;
+
+        if (!$this->tryReadJson($composerLockPath, $composerLock)) {
+            return false;
+        }
+
+        if (isset($composerLock['packages'])) {
+            $packages = $composerLock['packages'];
+            foreach ($packages as $package) {
+                if (!isset($package['name']) ||
+                    $package['name'] != "swedbank-pay/swedbank-pay-sdk-php") {
+                    continue;
+                }
+
+                if (isset($package['version'])) {
+                    $version = $package['version'];
+
+                    if (!is_null($version) && !empty($version)) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The use of function is_null() is discouraged; use strict comparison "=== null" instead.](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-sdk-php/pullRequest?prid=5312212)